AnnoyancesFilter/Other/sections/annoyances.txt

```diff
@@ -2883,6 +2883,7 @@ jianshu.com#$#body { overflow: auto !important; }
 jianshu.com##.dt-open-bg[data-scene="leftBtn"]
 jianshu.com##.note-comment-above-ad-wrap
 instiz.net#$?#.backlogo { background-image: none !important; }
+immersive-fox.com#@#.share-icon
```

Comment:
It was fixed directly here https://github.com/AdguardTeam/AdguardFilters/issues/134966.